### PR TITLE
Add support for Mockserver and, since required, for containers exposing multiple ports.

### PR DESCRIPTION
**Pull request by ibrt**
…ng multiple ports.

### CODE REVIEW

**Comment by arekkas** [comment]
Hey very cool stuff. I will need to check everything with a little bit of time because this has the potential for BC breaks. Maybe we need to release v2 if this get's merged.

**Comment by ibrt** [comment]
Thanks! I think there is one breaking change in the signature of SetupContainer (since it's public). If you prefer to do it in a backwards compatible way we could add a SetupContainerWithMultiplePorts function instead.

**Comment by arekkas** [comment]
That could be an option, too. I will check tomorrow and think about the best way to tackle this :) Thanks for the PR!

**Comment by arekkas** [comment]
Yes, this should be a new method, how about `SetupMultiportContainer`?

**Comment by ibrt** [comment]
Done :)

**Comment by arekkas** [comment]
Neat! Will audit and merge once i have wlan :)

Gesendet mit meinem HTC

----- Reply message -----
Von: "Ivan Bertona" <notifications@github.com>
An: "ory-am/dockertest" <dockertest@noreply.github.com>
Cc: "Aeneas" <aeneas.rekkas@serlo.org>
Betreff: [dockertest] Add support for Mockserver and, since required, for containers exposing multiple ports. (#32)
Datum: Mi., Mär. 9, 2016 00:14

Done :)

---

File: docker.go [hunk 1]
```diff
@@ -247,3 +254,7 @@ func RandomPort() int {
 func GenerateContainerID() string {
 	return ContainerPrefix + uuid.New()
 }
+
+func init() {
```
Comment (arekkas) [inline docker.go]: I couldn't find where this method is being called, did I miss it or you? :)

Comment (arekkas) [inline docker.go]: Oh wow, didn't know that init is always run in Go.

**Comment by arekkas** [comment]
Thanks!

**Pull request merged**